CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
   figure.
 * Fixed issue with Speck not rendering unless it is attached to a callback.
 * Prevent Speck from trying to calculate a system with no atom.
+* Prevent Clustergram from not rendering when optional row and/or column
+  labels have not been specified.
```

Comment:
```suggestion
  labels.
```
Let's be descriptive and positive rather than double negative (plus verb should be preterit) :)